<launch>
  <param name="use_sim_time" value="true" />

  <!-- Launch the world simulator, guiabot simulator, and simulated tag finder -->
  <include file="$(find abstract_map_simulator)/launch/world.launch" />
  <node pkg="abstract_map_simulator" type="find_tags_simulated" name="tag_finder_simulated" >
    <param name="sensor_range" value="5.0" />
    <param name="sensor_fov_deg" value="360" />
  </node>
</launch>
